docs/testnet/wallet-extension.md

```diff
@@ -32,7 +32,7 @@ unencrypted. If the data is not particularly sensitive, it can also be run in an
 
     * For other OSes and architectures, see `Compiling the binary`, below
 
-2. Start the wallet extension by running the `wallet_extension` binary with the following flags:
+2. Open a terminal window of command prompt. Start the wallet extension by running the `wallet_extension` binary with the following flags:
 
    ```--nodeRPCHTTPAddress=<Obscuro host RPC HTTP address> --nodeRPCWebsocketAddress=<Obscuro host RPC websocket address>```
```

Comment:
Can we give the testnet defaults here? Or do they rotate too often?

Reply:
I've flip flopped here myself but actually testnet.obscu.ro and the port numbers are likely to be fairly static. I'll change the text.